pkg/ccl/importccl/read_import_pgdump.go

```diff
@@ -488,6 +488,17 @@ func readPostgresStmt(
 		// ignore txns.
 	case *tree.SetVar, *tree.Insert, *tree.CopyFrom, copyData:
 		// ignore SETs and DMLs.
+	case *tree.Delete:
```

Comment:
Nit: I think I’d put `*tree.Delete` in the case above this to just fast-ignore it along with other DMLs,  since here is during schema extraction. Instead I’d move what you have in this case to a stand-alone `Delete` case in the data extraction pass below (where you have it unconditionally ignored)

Reply:
Done! Cheers for the late night peek.